### PR TITLE
Fb lineage graph testing

### PR DESCRIPTION
**Pull request by labkey-chrisj**
#### Rationale
This updates some test code to use new lineage graph component; we anticipate replacing the existing graphviz component with this newer one

#### Related Pull Requests
The new components are in testAutomation/fb_lineageGraphTesting
Similar changes are in provenance/fb_lineageGraphTesting

#### Changes
Where we have the new lineage graph component (on Experiment Run page) refactor tests to use that instead

### CODE REVIEW

File: ms2/test/src/org/labkey/test/ms2/AbstractMS2SearchEngineTest.java [hunk 1]
```diff
@@ -129,9 +134,23 @@ public void basicMS2Check()
...
+        Locator.linkWithSpan("Toggle Beta Graph (new!)").waitForElement(getDriver(), 4000)
+                .click();
+        LineageGraph graphComponent = new LineageGraph.LineageGraphFinder(getDriver()).waitFor();
+        List<String> actualParents = graphComponent.getDetailGroup("Data Parents")
+                .getItemNames();
+        List<String> actualChildren = graphComponent.getDetailGroup("Data Children")
+                .getItemNames();
+        assertThat(actualParents, hasItems("Bovine_mini1.fasta", "comet.xml", "CAexample_mini.mzXML"));
+        assertThat(actualChildren, hasItems("CAexample_mini.prot.xml", "CAexample_mini.pep.xml",
+                "CAexample_mini.mzXML.image..itms.png"));
```
Comment (labkey-tchad) [inline ms2/test/src/org/labkey/test/ms2/AbstractMS2SearchEngineTest.java]: Are you sure that these lineages are valid for all of the subclasses of `AbstractMS2SearchEngineTest`?
If you haven't, I would suggest removing these assertions instead of updating them. We should explicitly verify the run graph somewhere rather than doing a drive-by in an arbitrary test.

File: ms2/test/src/org/labkey/test/ms2/AbstractMS2SearchEngineTest.java [hunk 2]
```diff
@@ -129,9 +134,23 @@ public void basicMS2Check()
...
+
+        // navigate to the details page for CAexample_mini.mzXML.image..itms.png
+        graphComponent.getDetailGroup("Data Children")
+                .getItem("CAexample_mini.mzXML.image..itms.png")
```
Comment (labkey-tchad) [inline ms2/test/src/org/labkey/test/ms2/AbstractMS2SearchEngineTest.java]: Use `SAMPLE_BASE_NAME` where able, to match the rest of the test.
```suggestion
                .getItem(SAMPLE_BASE_NAME + ".mzXML.image..itms.png")
```

File: ms2/test/src/org/labkey/test/ms2/QuantitationTest.java [hunk 3]
```diff
@@ -83,7 +84,11 @@ public void testSteps()
         // Jump to the flow chart view
         clickAndWait(Locator.tagWithAttribute("a", "title", "Experiment run graph"));
 
-        clickAndWait(Locator.imageMapLinkByTitle("graphmap", "Data: " + SAMPLE_BASE_NAME + ".libra.tsv (Run Output)"));
+        Locator.linkWithSpan("Toggle Beta Graph (new!)").waitForElement(getDriver(), 4000)
+                .click();
+        LineageGraph graph = new LineageGraph.LineageGraphFinder(getDriver()).waitFor();
+
+        graph.getDetailGroup("Data Children").getItem("CAexample_mini.libra.tsv").clickOverViewLink(true);
```
Comment (labkey-tchad) [inline ms2/test/src/org/labkey/test/ms2/QuantitationTest.java]:
```suggestion
        graph.getDetailGroup("Data Children").getItem(SAMPLE_BASE_NAME + ".libra.tsv").clickOverViewLink(true);
```

**Review by labkey-tchad — CHANGES_REQUESTED** [review]

**Comment by labkey-tchad** [comment]
@labkey-chrisj 
Regarding the "**Related Pull Requests**" section of the template; please provide actual links to the related pull requests. GitHub will even automatically format them all nice for you:
https://github.com/LabKey/testAutomation/pull/314
https://github.com/LabKey/provenance/pull/17

File: ms2/test/src/org/labkey/test/ms2/AbstractMS2SearchEngineTest.java (hunk 1 above)
Comment (labkey-chrisj) [reply ms2/test/src/org/labkey/test/ms2/AbstractMS2SearchEngineTest.java]: good call; we don't need to insert this into all of them

**Review by labkey-tchad — APPROVED** [review]
